src/main/java/de/tum/in/www1/artemis/service/dataexport/DataExportExamCreationService.java

```diff
@@ -61,14 +66,22 @@ public void createExportForExams(long userId, Path workingDirectory) throws IOEx
                 var exam = studentExam.getExam();
                 var examTitle = exam.getSanitizedExamTitle();
                 var courseDirPath = retrieveCourseDirPath(workingDirectory, exam.getCourse());
-                createDirectoryIfNotExistent(courseDirPath);
+                var examsDirPath = courseDirPath.resolve("exams");
+                createDirectoryIfNotExistent(examsDirPath);
                 var examDirectoryName = EXAM_DIRECTORY_PREFIX + examTitle + "_" + studentExam.getId();
-                var examWorkingDir = Files.createDirectories(courseDirPath.resolve(examDirectoryName));
+                var examWorkingDir = Files.createDirectories(examsDirPath.resolve(examDirectoryName));
```

Comment:
Is this a doubled directory creation? Once in createDirectoryIfNotExistent and again in createDirectories?